### ISSUE TITLE
Consider conformance of HybridSign static text

### ISSUE THREAD

**Issue by j-d-b** [usdot-jpo-ode/wzdx#275]
## Background

Currently, the `static_sign_text` property on the [HybridSign](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/HybridSign.md) object is optional. The property is described as:

> The static text on the non-electronic component of the hybrid sign.

## Issue

Members have asked several times why the `static_sign_text` was optional, and I couldn't find an answer in the specification documentation or my meeting notes. This issue seeks to start discussion that could lead to requiring this property in a future release.

## Discussion

- Discuss if the HybridSign `static_sign_text` property should be required.

**Comment by sergebeaudry** [comment]
I agree that this could be required. I only see cases where some text exist on those. Speed Limit, Travel Time, height limit, width limit, etc.......
 Even the `dynamic_message_text` might be required with potential to call it blanked

**Comment by j-d-b** [comment]
Based on SWZ subgroup co-chairs discussion, `static_sign_text` should be required. Thus the proposed change for v4.1 is to indicate in the documentation that the property will be required in a future release.

This issue will remain open until v5.0, when the property can be required.

**Comment by tfoster-vm** [comment]
Could it stay open for being required in v5.0 and also be added still be added as optional until v5.0?
